lib/input.rb

```diff
@@ -1,5 +1,10 @@
 class Input
-    def get_input
-        $stdin.gets.chomp.upcase
+    def get_num_players
+        $stdin.gets.chomp
     end
+
+    def get_player_token
+        $stdin.gets.chomp
+    end
```

Comment:
Both of these functions do the same thing.

Reply:
I will delete this.